packages/eslint-config-sheriff/src/getBaseConfig.ts

```diff
@@ -138,6 +136,7 @@ export const getBaseConfig = (userConfigChoices: SheriffSettings) => {
       settings: {
         'import/parsers': {
           '@typescript-eslint/parser': ['.ts', '.tsx'],
+          espree: ['.js'],
```

Comment:
what is this for?

Reply:
The linked issue said to add it, so I did. 🤷‍♂️

Reply:
I think this won't hurt, even though w don't really support .js files.